### PR TITLE
fix to deleting metadata in elasticsearch

### PR DESCRIPTION
**Pull request by ddey2**

### CODE REVIEW

**Comment by tcnichol** [comment]
An error comes up when deleting metadata but there is another pull request for that bug:f
https://github.com/clowder-framework/clowder2/pull/245

With those fixes this pull request works and properly deletes from elasticsearch. Should I mark this one approved, or wait until that one is merged into main and then main is merged in here?

**Comment by ddey2** [comment]
@tcnichol I reviewed and approved the #245 
I believe merging both should be good.

**Review by lmarini — APPROVED** [review]

**Pull request merged**
